src/xdocs/config_coding.xml

```diff
@@ -1221,6 +1221,32 @@ case 5:
          }
          </source>
          </p>
+         <p>
+         To configure the check so that it checks local variables and parameters and it also validates enhanced For Loop Variable is:
```

Comment:
please split this long line in one, to keep all text below 100 symbols,
Sorry that this is not enforced yet by build, There is issue on this, it is in progress.